components/Taxation/basic_usage.rst

```diff
@@ -0,0 +1,179 @@
+Basic Usage
+===========
+
+Tax Rate
+--------
+
+Every tax rate has two identifiers, an ID and name. You can access those by calling ``->getId()`` and ``getName()``
+respectively. The name is mutable, so you can change it by calling ``->setName('EU VAT')`` on the tax rate instance.
+
+Tax Amount
+~~~~~~~~~~
+
+An tax rate has two basic amounts. The first is the *amount* and the second is *amount as percentage* (default it is set to 0).
```

Comment:
**A** tax rate has two basic amounts - the*amount* and the *amount as percentage* (by default equal 0).